util/Migrator/DbScripts/2022-08-10_00_Project.sql

```diff
@@ -0,0 +1,15 @@
+IF OBJECT_ID('[dbo].[Project]') IS NULL
+BEGIN
+CREATE TABLE [dbo].[Project] (
+    [Id]                UNIQUEIDENTIFIER NOT NULL,
+    [OrganizationId]    UNIQUEIDENTIFIER NOT NULL,
+    [Name]              NVARCHAR(MAX) NULL, 
+    [CreationDate]      DATETIME2 (7),
+    [RevisionDate]      DATETIME2 (7), 
+    [DeletedDate]       DATETIME2 (7) NULL,
+    CONSTRAINT [PK_Project] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_Project_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]),
```

Comment:
Same thing here with the `ON DELETE CASCADE`

Reply:
Thank you! Updated